google-http-client/src/main/java/com/google/api/client/http/javanet/NetHttpTransport.java

```diff
@@ -275,6 +293,32 @@ public Builder trustCertificates(KeyStore trustStore) throws GeneralSecurityExce
       return setSslSocketFactory(sslContext.getSocketFactory());
     }
 
+    /**
+     * Sets the SSL socket factory based on a root certificate trust store and a client certificate
+     * key store.
+     *
+     * @param trustStore certificate trust store (use for example {@link SecurityUtils#loadKeyStore}
+     *     or {@link SecurityUtils#loadKeyStoreFromCertificates})
+     * @param keyStore key store for client certificate and key
+     * @param keystorePassword password for keyStore parameter
+     */
+    public Builder trustCertificates(
```

Comment:
Builder setters should be `setSomething()` or possibly `applySomething()`. The documentation should also indicate that this affects MTLS configuration.

Reply:
This file has a couple of `trustCertificates` like methods to set SSL socket. I am using the same name to be consistent with those methods.

Documentation is updated.